packages/icons/src/library/group.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const group = (
+	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+		<Path fillRule="evenodd" clipRule="evenodd" d="M9 8a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1v4a1 1 0 0 1-1 1h-1v3a1 1 0 0 1-1 1H8a1 1 0 0 1-1-1v-4a1 1 0 0 1 1-1h1V8zm2 3h4V9h-4v2zm2 2H9v2h4v-2z" />
```

Comment:
I imagine we can remove the attributes `fillRule` and `clipRule` both. Worth viewing the SVG to verify, but I believe they are properties that are unused because this SVG is neither clipped nor filled.

Reply:
Removing these actually break the SVG

Reply:
Alright, that was a chance. Thanks for testing, and then keep them in.